README.md

```diff
@@ -7,6 +7,10 @@ ACS Reports Team test project.
 - Install required engines with `nvm install`.
 - Install dependencies with `npm install`
 - Run `docker-compose up -d db`. The postgres instance will be attached to port 5400
+- Make sure you have the `postgresql` client installed locally by running `which createdb`.
+  - You can install `createdb` via `homebrew` by running `brew install libpq`
+  or with `apt install postgresql-client` in Windows Linux Subsystems or Debian-like
+  installations.
```

Comment:
This doesn't install the full `postgresql` database, it only installs the client libraries such as `psql` and `createdb`